hadoop-ozone/dist/src/main/compose/ozone/docker-config

```diff
@@ -48,3 +48,5 @@ OZONE_CONF_DIR=/etc/hadoop
 OZONE_LOG_DIR=/var/log/hadoop
 
 no_proxy=om,scm,s3g,recon,kdc,localhost,127.0.0.1
+
+export SECURITY_ENABLED=false
```

Comment:
Thank you for replying.
For some reason, when running the tests without setting `export SECURITY_ENABLED` in the docker-config, the S3 tests would fail with the error `Environment variable 'SECURITY_ENABLED' does not exist.`

Reply:
I'm not sure why that happens, but it should not be read from environment variable, see my [comment](https://github.com/apache/ozone/pull/3629#discussion_r947005888).